app/src/main/java/com/monzo/androidtest/individual/presentation/IndividualState.kt

```diff
@@ -0,0 +1,13 @@
+package com.monzo.androidtest.individual.presentation
+
+
+
+data class IndividualState(
+    val article: IndividualArticle? = null
```

Comment:
Evite default values

```suggestion
    val article: IndividualArticle?
```